### PR TITLE
Fix quality control issue of chemical DA

### PR DESCRIPTION
**Pull request by liujake**
TYPE: bug fix

KEYWORDS: Quality control, chemical DA

SOURCE: Jake Liu (NCAR/MMM)

DESCRIPTION OF CHANGES:
Problem:
PR #1575 introduced a new chemical DA option for the MADE-VBS aerosol scheme (chem_cv_options=108).
Ideally this should not change the observation part of the chemical DA code. However, current implementation also modified observation part along with the new option (e.g., obs file format, obs units, quality control).  The New QC check for rejecting obs with omb>maxomb leads to too many obs rejected for the obs file format for other chemical DA options originally implemented in v4.3. 

Solution:
Restrict the new maxomb QC check only for chem_cv_options=108. Also limit a print statement only for chem_cv_options=108. This does not recover exactly the v43 QC for other options, but it is close.

LIST OF MODIFIED FILES:
M       var/da/da_chem_sfc/da_get_innov_vector_chem_sfc.inc
M       var/da/da_obs_io/da_read_obs_chem_sfc.inc

TESTS CONDUCTED: 
The statistic_chem file now has the similar number of good obs for chem_cv_options=20 (case provided in the user's guide) with the fixed code. 

RELEASE NOTE: None.

### CODE REVIEW

**Comment by weiwangncar** [comment]
The Jenkins tests have passed:

    Test Type              | Expected  | Received |  Failed
    = = = = = = = = = = = = = = = = = = = = = = = =  = = = =
    Number of Tests        : 23           24
    Number of Builds       : 60           58
    Number of Simulations  : 158           156        0
    Number of Comparisons  : 95           92        0

    Failed Simulations are: 
    None
    Which comparisons are not bit-for-bit: 
    None

**Comment by syha** [comment]
For clarification -
1) the observation part of the chemical DA code was highly customized for specific dataset, which did not work for my data from different resources, so I had to make changes. At some point, it would be nice to provide a preprocessing tool to generalize the input chem data format. For now, we need to go with the if-statement for different chem_opt because the test data for each option comes with different format and units.
2) DA algorithms do need QCs based on omb to minimize the shock to the model. This is a reliability issue, and has nothing to do with chemical options. So we should not limit the QCs to a specific chem_opt, as suggested.
Instead, I propose to relax maxomb values to (/300, 500, 2, 2, 2, 2/) in da_get_innov_vector_chem_sfc.inc. (Feel free to make them larger, if you want.) 
As we are about to release the new version, we cannot make any major changes now, but for the next release, we can certainly put them in the namelist to let users adjust the values as they want. But we should keep this capability regardless of chem_opt.
In a nutshell, I suggest you revert all the changes you made here.

**Comment by liujake** [comment]
@syha will make another PR soon for this. closing this one.
